source/includes/starting-compass-paste-string.rst

```diff
@@ -28,7 +27,20 @@
       .. include:: /includes/fact-hide-connection-string-password.rst
             
       To learn how to format the connection string for a deployment that is not 
-      hosted on |service|, see :manual:`Connection String URI Format </reference/connection-string>`.
+      hosted on |service|, see :manual:`Connection String URI Format
+      </reference/connection-string>`.
+      
+   .. step:: (Optional). Name your connection.
+
+      Use the name field to enter a name for your connection. If you do
+      not specify a name, Compass uses the cluster's hostname as the the
+      connection name. 
+
+   .. step:: (Optional). Choose a color for your connection.
+
+      Use the color drop-down menu to select a label color for your
+      connection. When you activate a connection, the label color is the
```

Comment:
Are we going with "activate a connection"? Isn't it just connect and disconnect?

Reply:
I used "activate" based on the "Active Connections" menu but changed to "connect"